testsuites/store/src/main/java/org/eclipse/rdf4j/sail/InferencingTest.java

```diff
@@ -49,7 +49,7 @@ public void runTest(Sail sailStack, String subdir, String testName, boolean isPo
 		final String inputData = TEST_DIR_PREFIX + "/" + name + "-in.nt";
 		final String outputData = TEST_DIR_PREFIX + "/" + name + "-out.nt";
 
-		Collection<? extends Statement> entailedStatements = null;
+		Collection<? extends Statement> entailedStatements = new HashSet<>();
```

Comment:
I remember why I added this. There is a try-catch around the code that sets entailedStatements, and the catch block silences the exception. Then later on in the code, the entailedStatements is checked against the expected statements without null checking entailedStatements causing a null pointer exception.

I'll add in e.printStackTrace(); and also leave the entailedStatements as non null. Otherwise this test is very hard to debug when something fails.